sdk/ai/ai-inference-rest/README.md

```diff
@@ -545,45 +541,53 @@ provider.register();
 
 To use instrumentation for Azure SDK, you need to register it before importing any dependencies from `@azure/core-tracing`, such as `@azure-rest/ai-inference`.
 
-```js
+```ts snippet:ReadmeSample_InstrumentationImport
 import { registerInstrumentations } from "@opentelemetry/instrumentation";
 import { createAzureSdkInstrumentation } from "@azure/opentelemetry-instrumentation-azure-sdk";
 
 registerInstrumentations({
   instrumentations: [createAzureSdkInstrumentation()],
 });
-
-import ModelClient from "@azure-rest/ai-inference";
 ```
 
-Finally when you are making a call for chat completion, you need to include
+Finally when you are making a call for chat completion, you need to include the `tracingOptions` in the request. Here is an example:
 
-```js
-tracingOptions: {
-  tracingContext: context.active();
-}
-```
+```ts snippet:ReadmeSample_InstrumentationRequest
+import { DefaultAzureCredential } from "@azure/identity";
+import ModelClient from "@azure-rest/ai-inference";
+import { context } from "@opentelemetry/api";
 
-Here is an example:
+const endpoint = "<your_model_endpoint>";
```

Comment:
Should this also be "https://myaccount..."?

Reply:
fixing across the board.